### DISCUSSION TITLE
Upgrade Depth Anything V2 Models?

### DISCUSSION THREAD

**Discussion by diverswan9** [nagadomi/nunif#182] [Q&A] [answered]
A Question... should we periodically be downloading Depth Anything V2 Models from the repo? Mine that I manually downloaded (Anything V2) I got back when they first became available. Like, so if I download them now are they 'upgraded' ?

**Comment by nagadomi** [comment, marked as answer]
I am not admin of Depth-Anything repo, so I don't know if it gets updated or not.
Basically, I expect that if something changes, it will be rereleased as a different file.
Also I checked Depth-Anything repo and it has not been updated since the first release (Jun 14) .

https://huggingface.co/depth-anything/Depth-Anything-V2-Base/tree/main
https://huggingface.co/depth-anything/Depth-Anything-V2-Large/tree/main

**Reply by diverswan9** [reply]
Thank you.
